py/desimodel/io.py

```diff
@@ -777,6 +777,6 @@ def datadir(surveyops=False):
         if "DESIMODEL" in os.environ:
             return os.path.abspath(os.path.join(os.environ["DESIMODEL"], "data"))
         else:
-            import pkg_resources
+            import importlib
 
-            return pkg_resources.resource_filename("desimodel", "data")
+            return importlib.resources.files("desimodel").joinpath("data")
```

Comment:
This is technically an API change because `resource_filename()` returns a string, while `files()` returns a `Path`. Also note that if `DESIMODEL` is defined (line 777), this function will return a string. So you have potentially inconsistent return types.

Reply:
Good point.  I'll wrap it in `str(...)` so that this function always returns a string like before, whether $DESIMODEL is set or not.